tools/migration/main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"runtime"
+
+	"github.com/filecoin-project/go-filecoin/tools/migration/internal"
+	"github.com/filecoin-project/go-filecoin/util/version"
+)
+
+func main() { // nolint: deadcode
+	if len(os.Args) < 2 {
+		showUsageAndExit(1)
+	}
+
+	if !version.Check(runtime.Version()) {
+		exitErr(fmt.Sprintf("Invalid go version: %s", runtime.Version()))
+	}
+
+	command := getCommand()
+
+	switch command {
+	case "-h", "--help":
+		showUsageAndExit(0)
+	case "describe", "buildonly", "migrate", "install":
+		mr := internal.MigrationRunner{}
+		if err := mr.Run(command, getVerbose()); err != nil {
+			exitErr(err.Error())
+		}
+	default:
+		exitErr(fmt.Sprintf("Error: Invalid command: %s", command))
+	}
+}
+
+// TODO: should this go to the same logs as the migration log? Maybe not.
```

Comment:
my .02 is that we should log all errors to the same place until we have a compelling reason to get more complicated.